src/components/phone/ContactMoreInfo.vue

```diff
@@ -6,26 +6,58 @@
     <div class="info">
       <div class="flex flex-col">
         <div class="justify-between flex flex-row m-3">
-          <base-text variant="h4" class="text-crisiscleanupgrey-200"
-            >Languages</base-text
-          >
-          <base-text variant="h4" class="text-crisiscleanupgrey-200"
-            >eng</base-text
-          >
+          <div class="flex flex-col w-1/2">
+            <base-text
+              variant="h4"
+              weight="300"
+              class="text-crisiscleanup-grey-800"
+              >Languages</base-text
+            >
+          </div>
+          <div class="flex flex-col">
+            <div class="align-left">
+              <div class="flex flex-row">
+                <div class="flex flex-col px-2">
+                  <tag
+                    class="text-crisiscleanup-red-300 border-crisiscleanup-red-300"
+                    >English</tag
+                  >
+                </div>
+                <div class="flex flex-col px-2">
+                  <tag
+                    class="text-crisiscleanup-dark-blue border-crisiscleanup-dark-blue"
+                    >Spanish</tag
+                  >
+                </div>
+              </div>
+            </div>
+          </div>
         </div>
         <div class="justify-between flex flex-row m-3">
-          <base-text variant="h4" class="text-crisiscleanupgrey-200"
-            >Counties</base-text
-          >
-          <base-text class="text-sm">Texas, Louisiana</base-text>
+          <div class="flex flex-col w-1/2">
+            <base-text
+              variant="h4"
+              weight="300"
+              class="text-crisiscleanup-grey-800"
+              >Counties</base-text
+            >
+          </div>
+          <div class="flex flex-col w-1/2">
+            <base-text class="text-sm">Texas, Louisiana</base-text>
+          </div>
         </div>
-        <div class="justify-between flex flex-row m-3">
-          <base-text variant="h4" class="text-crisiscleanupgrey-200"
-            >My Availability</base-text
-          >
-          <base-button class="text-crisis-cleanup-yellow-900 align-right"
-            >Show</base-button
-          >
+        <div class="flex flex-row m-3">
+          <div class="flex flex-col w-1/2">
+            <base-text
+              variant="h4"
+              class="text-crisiscleanup-grey-800"
+              weight="300"
+              >My Availability</base-text
```

Comment:
So all this text needs to be translated by @aarontitus , to make this easier on him, we started doing something this this:

```js
.......
  computed: {
    lang() {
      return {
        title: this.$t('~~Welcome to Crisis Cleanup Phone Center'),
        action: {
          title: this.$t(''),
          button: this.$t('~~Take me to the pop-up page'),
          user_id: this.$t('~~Username: '),
          password: this.$t('~~Password: '),
        },
        copy: {
          button: this.$t('~~Copy'),
          successMessage: this.$t('~~Copied!'),
        },
      };
    },
  }
.......

```

the `this.$t(...)` function takes a key (like: `actions.popup`) which then fetches the correct text in the users language.

Since you don't have the keys (and they may not exist yet), you need to add a lang() element like this (the object inside can be layed out however is appropriate to your component) and then put your filler text in preceded with '~~'

The double tildes are for @aarontitus , so he can easily go in and replace our filler text with the correct translation key.

So as a quick example, you may end up with something like this:

_ContactMoreInfo.vue_

```js

...
<some-element>
  {{ lang.action.show }}
</some-element>
...


...
computed: {
    lang() {
      return {
        action: {
	  show: this.$t('~~Show'),
	},
      	lang: { 
	  key: this.$t('~~Languages') 
	  value: [this.$t('~~English'), this.$t('~~Spanish')] // use the v-for bind to iterate these
	}
      };
    },
  }
...
```